grore/settings.py

```diff
@@ -27,15 +37,20 @@
 SECRET_KEY = os.environ.get("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = bool(os.environ.get("DEBUG", default=0))
+if os.environ.get("DEBUG", 'false').lower == 'true':
```

Comment:
This does not work. I have a corrected version now in my development after I have merge it with the present current version of integration.